<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManageWindow</class>
 <widget class="QMainWindow" name="ManageWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>10</y>
      <width>300</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe 仿宋 Std R</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>COVID-19 疫情行程风险管理系统</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>751</width>
      <height>441</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="addBtn">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>530</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>添加此信息</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ingnoreBtn">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>530</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>忽略此信息</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>班次编辑</string>
    </property>
    <addaction name="shwAct"/>
    <addaction name="addAct"/>
    <addaction name="delAct"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>风险管控</string>
    </property>
    <addaction name="shwrskAct"/>
    <addaction name="addrskAct"/>
    <addaction name="delrskAct"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>操作</string>
    </property>
    <addaction name="retAct_2"/>
    <addaction name="exitAct_2"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <action name="addAct">
   <property name="text">
    <string>Add Traffic</string>
   </property>
  </action>
  <action name="delAct">
   <property name="text">
    <string>Delete Traffic</string>
   </property>
  </action>
  <action name="exitAct">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="retAct">
   <property name="text">
    <string>Return</string>
   </property>
  </action>
  <action name="retAct_2">
   <property name="text">
    <string>Return</string>
   </property>
  </action>
  <action name="exitAct_2">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="addrskAct">
   <property name="text">
    <string>Add Risk</string>
   </property>
  </action>
  <action name="delrskAct">
   <property name="text">
    <string>Del Risk</string>
   </property>
  </action>
  <action name="shwAct">
   <property name="text">
    <string>Show Traffic</string>
   </property>
  </action>
  <action name="shwrskAct">
   <property name="text">
    <string>Show Risk</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
